docs/tool/manifest.js

```diff
@@ -51,7 +51,8 @@ function getPackageManifest( packageFolderNames ) {
  */
 function getComponentManifest( paths ) {
 	return paths.map( ( filePath ) => {
-		const slug = nth( filePath.split( '/' ), -2 );
+		const pathFragments = filePath.split( '/' );
+		const slug = pathFragments[ pathFragments.length - 2 ];
```

Comment:
I think there might be an issue here. The offset of `_.nth` is using index 0 whiles `.length` is using index 1. So this would need to be -3 instead of -2.

(might be wrong. just want to make sure)

Reply:
Thanks for double-checking, @fabiankaegy. Actually, Lodash uses 1-based indexes here, as well.

To confirm, give this a try:
* Go to https://lodash.com/docs/4.17.15
* In the browser console, type:

```
> const arr = [ 1, 2, 3, 4, 5 ];
> console.log( arr[ arr.length - 2 ] );
> console.log(_.nth( arr, -2 ));
```
* You should get `4` for both `console.log()` calls.